django_walletpass/classviews.py

```diff
@@ -1,24 +1,24 @@
 import json
 from calendar import timegm
 
-import django.dispatch
 from dateutil.parser import parse, ParserError
 from django.db.models import Max
 from django.http import HttpResponse
 from django.middleware.http import ConditionalGetMiddleware
 from django.shortcuts import get_object_or_404
 from django.utils.http import http_date
+from pytz.exceptions import NonExistentTimeError
```

Comment:
Since we already require >=py3.10, I'd rather avoid using pytz (which is already in maintenance mode).

Reply:
done: d7ecb97